### PR TITLE
Bigdime Management Console changes

### PR DESCRIPTION
**Pull request by sandeepreddymurthy1**

### CODE REVIEW

File: bigdime-management-console/angularjs-client/app/js/app.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+ * @name jsonerApp
+ * @description
+ * # jsonerApp
+ *
+ * Main module of the application.
+ */
+angular
+.module('jsonerApp', ['ui.router'])
+.constant("AdaptorConstants",[{
+	"environment":"Dev",
```
Comment (a-ninja) [inline bigdime-management-console/angularjs-client/app/js/app.js]: The handlers shouldn't he hardcoded here, they should be fetched from a data source, be it another service or a datastore etc.

File: bigdime-management-console/angularjs-client/app/js/controllers/homecontroller.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/**
+ * @author Sandeep Reddy,Murthy
```
Comment (mnamburi) [inline bigdime-management-console/angularjs-client/app/js/controllers/homecontroller.js]: add a copyright for all the files.
Copyright (C) 2015 Stubhub.

File: bigdime-management-console/angularjs-client/app/js/services/ApplicationService.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+			applicationService.setEnvConstants=setEnvConstants;
+			applicationService.getEnvConstants=getEnvConstants;
+			applicationService.envConstants=[];
+			return applicationService;
+            
+			function setEnvConstants(){
+				var envDetails={};
+				$.ajax({
+					  dataType: "json",
+					  url:'rest/propertyService/applicationproperties',
```
Comment (mnamburi) [inline bigdime-management-console/angularjs-client/app/js/services/ApplicationService.js]: rest/v1? can you add version ?

File: bigdime-management-console/angularjs-client/app/views/adaptorsselection/fileadaptorselection.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<form class="padding-top">
+	<div class="form-group row">
+		<label class="col-xs-3">Source Input</label>
```
Comment (mnamburi) [inline bigdime-management-console/angularjs-client/app/views/adaptorsselection/fileadaptorselection.html]: can you get rid of the 3rd party names in your project structure unless you treat them as a library.

management-console/angularjs-client/app/views/adaptorsselection/fileadaptorselection.html

**Pull request merged**